addons/binding/org.openhab.binding.verisure/README.md

```diff
@@ -0,0 +1,359 @@
+| location        | String    | This channel reports the location.                                          |
+| siteName        | String    | This channel reports the name of the site.                                  |
+| siteId          | Number    | This channel reports the site ID of the site.                               |
+
+### Verisure User Presence
+
+#### Configuration Options
+
+*   deviceId - Device Id
+    *   Since User presence lacks a Verisure ID, the following naming convention is used for User presence on first site: 'userpresence_1'
+
+#### Channels
+
+([userPresence]) supports the following channels:
+ 
+| Channel Type ID    | Item Type | Description                                                 | 
+|--------------------|-----------|-------------------------------------------------------------|                                                                                                                                          
+| webAccount         | String    | This channel reports the user's email.                      |
+| userLocationStatus | String    | This channel reports the user presence (HOME/AWAY).         |
+| userLocationName   | String    | This channel reports the name of the location (can be null).|
+| siteName           | String    | This channel reports the name of the site.                  |
+| siteId             | Number    | This channel reports the ID of the site.                    |
+
+### Verisure Broadband Connection
+
+#### Configuration Options
+
+*   deviceId - Device Id
+    *   Since Broadband connection lacks a Verisure ID, the following naming convention is used for Broadband connection on first site: 'broadband_1'
+
+#### Channels
+
+([broadbandConnection]) supports the following channels:
+ 
+| Channel Type ID | Item Type | Description                                                                    | 
+|-----------------|-----------|--------------------------------------------------------------------------------|                                                                                                                                          
+| timestamp       | String    | This channel reports the last time the Broadband connection status was checked.|
+| hasWiFi         | String    | This channel reports if user has WiFi connection.                              |
+| status          | String    | This channel reports the broadband connection status.                          |
+| siteName        | String    | This channel reports the name of the site.                                     |
+| siteId          | Number    | This channel reports the ID of the site.                                       |
+
+## Example
+
+### Things-file
+
+````
+// Bridge configuration
+Bridge verisure:bridge:myverisure "Verisure Bridge" [username="x@y.com", password="1234", refresh="600", pin="111111", numberOfInstallations="1"]
```

Comment:
Things should be belong to one bridge, so correct way is

Bridge verisure:bridge:myverisure "Verisure Bridge" [username="x@y.com", password="1234", refresh="600", pin="111111", numberOfInstallations="1"] {
	
	Thing smartLock JannesSmartLock "Verisure Entrance Yale Doorman"  [ deviceId="3C44_6NPO" ]
	Thing smartPlug JannesSmartPlug "Verisure SmartPlug" [ deviceId="3D7G_MANV" ]
	Thing waterDetector JannesVattenDetetktor "Verisure Water Detector" [ deviceId="3WET_QRH5" ]
}

Otherwise binding doesn't know which bridge to use if someone have multiple bridges.

Reply:
Good spotting! I'll fix!